src/main/resources/db/migration/v10__client_avg_value_project.sql

```diff
@@ -0,0 +1,5 @@
+Select c.name as 'Client Name', AVG(p.value) as 'Average Project Value'
+from Client c
+join Project as p using (client_id)
+group by c.name
+order by AVG(p.value) LIMIT 1;
```

Comment:
why is there a limit here if its asking for average client values of all clients?